dataeng/jobs/analytics/SnowflakeQueueDepth.groovy

```diff
@@ -0,0 +1,73 @@
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+
+
+class SnowflakeQueueDepth {
+
+    public static def job = { dslFactory, allVars ->
```

Comment:
Where is allVars coming in from, and how do I define it? This should be where I define my environment variables, right?